Tools/Scripts/libraries/webkitbugspy/webkitbugspy/mocks/data.py

```diff
@@ -70,6 +70,7 @@
                 content='Trying to prepare a fix for 1.2.4',
             ),
         ], watchers=[USERS['Felix Filer'], USERS['Tim Contributor'], USERS['Wilma Watcher']],
+        references=[]
```

Comment:
Is this not part of what forms the `data` in `Tools/Scripts/libraries/webkitbugspy/webkitbugspy/mocks/bugzilla.py`? Because we're looking at `data['see_also']['add']` there.

Reply:
Also, not sure if I'm misunderstanding your comment, but the `data` you're referencing there is not the data from this file but rather from the formed request in `Bugzilla.set`

Reply:
Ah, OK, so I just don't understand what's going on here.